components/automate-ui/src/app/entities/service-groups/service-groups.effects.ts

```diff
@@ -103,4 +109,42 @@ export class ServiceGroupsEffects {
           new GetServiceGroupsSuggestionsSuccess({ serviceGroupsSuggestions })),
         catchError((error) => of(new GetServiceGroupsSuggestionsFailure(error))));
       }));
+
+  @Effect()
+  deleteServicesById$ = this.actions$.pipe(
+    ofType(ServiceGroupsActionTypes.DELETE_SERVICES_BY_ID),
+    mergeMap((action: DeleteServicesById) =>
+      this.requests.deleteServicesById(action.payload.servicesToDelete)),
+    map((response: GroupServicesPayload) =>
+      new DeleteServicesByIdSuccess({ amount: response.services.length })),
+    catchError((error: HttpErrorResponse) => of(new DeleteServicesByIdFailure(error))
+  ));
+
+  @Effect()
+  deleteServicesByIdFailure$ = this.actions$.pipe(
+    ofType(ServiceGroupsActionTypes.DELETE_SERVICES_BY_ID_FAILURE),
+    map(() => {
+      return new CreateNotification({
+        type: Type.error,
+        message: 'Unable to delete service. Please try again, or contact Chef Support.'
+      });
+    })
+  );
+
+  @Effect()
+  deleteServicesByIdSuccess$ = this.actions$.pipe(
+    ofType(ServiceGroupsActionTypes.DELETE_SERVICES_BY_ID_SUCCESS),
+    mergeMap((payload: any) => {
+      const amount = payload.payload.amount;
```

Comment:
@tarablack01 - This the area I had mentioned that I think i nested the payload somewhere, but i'm not totally sure how its happening

Reply:
also - the `any` type because of my nested payload right now

Reply:
nvm on this one, figured it out 👍